spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LeakyReLU.scala

```diff
@@ -32,12 +34,15 @@ import scala.reflect.ClassTag
  *                using extra state memory
  */
 
+
 @SerialVersionUID(- 6870619109313859155L)
 class LeakyReLU[T: ClassTag](
   private val negval: Double = 0.01,
   var inplace: Boolean = false)(
-  implicit ev: TensorNumeric[T]) extends TensorModule[T] {
+
+  implicit ev: TensorNumeric[T]) extends TensorModule[T] with IdentityOutputShape {
   import LeakyReLU._
+  private val negVal = ev.fromType[Double](negval)
```

Comment:
Why do you add this?

Reply:
Someone else add this. There's a conflict so I fix the conflict.

Reply:
Can you have a look in the latest code of this class? I don't think this line is still there.